sgv2-graphqlapi/src/main/resources/config.yaml

```diff
@@ -0,0 +1,10 @@
+server:
+  type: sgv2-graphql-service
+  applicationContextPath: /
+  connector:
+    type: http
+    port: 8083
```

Comment:
Isn't this the restapi new port?

Reply:
Yeah I think that's a copy-paste error, I'll use 8080 like v1.

Reply:
but how would you be able to start the Stargate and new GraphQL at the same time on the dev machine? Even int tests could have issues?

Reply:
Only if some other service tries to listen to port 8080. And given that it is existing port number, it must have already worked ok doesn't it? REST uses 8082, Auth api something else, CQL uses something else.

Reply:
That's what I am saying `v1/graphql` also serves at `8080`..